packages/core/scripts/util/android.ts

```diff
@@ -95,7 +98,7 @@ export const mainApplicationPatchable = ({
 
     if (!mainApplicationPath) {
       return reject(
-        `cannot find MainApplication.java file in any folder at ${p}. Please refer to the docs at https://embrace.io/docs/react-native/integration/add-embrace-sdk/?rn-platform=android to update it manually.`
+        `cannot find ${MAIN_CLASS_BY_LANGUAGE[platform]} file in any folder at ${p}. Please refer to the docs at https://embrace.io/docs/react-native/integration/add-embrace-sdk/?rn-platform=android to update it manually.`
```

Comment:
looks like the query string needs to be `?platform=android` to select the correct tab